RELEASE_NOTES.md

```diff
@@ -136,6 +167,30 @@ The `ChefModernize/FoodcriticComments` cop will remove any # ~FCXXX code comment
 
 `Autocorrects`: Yes
 
+#### ChefModernize/ExecuteScExe
+
+The `ChefModernize/ExecuteScExe` resource detects cookbooks that use the `execute` resource to run `sc.exe` in order to manage Windows services. Since Chef Infra Client 14.0 the `windows_service` resource has included the `:create`, `:delete`, and `:configure` actions for idempotently managing services.
+
+`Enabled by default`: False
+
+`Autocorrects`: No
+
+#### ChefModernize/WindowsScResource
+
+The `ChefModernize/ExecuteScExe` resource detects cookbooks that use the `sc_windows` resource from the `sc` coookbook in order to manage Windows services. Since Chef Infra Client 14.0 the `windows_service` resource has included the `:create`, `:delete`, and `:configure` actions which manage services without the need for additional dependencies.
```

Comment:
```suggestion
The `ChefModernize/ExecuteScExe` resource detects cookbooks that use the `sc_windows` resource from the `sc` coookbook in order to manage Windows services. Since Chef Infra Client 14.0, the `windows_service` resource has included the `:create`, `:delete`, and `:configure` actions which manage services without the need for additional dependencies.
```